core/src/test/java/com/linecorp/armeria/common/FlagsTest.java

```diff
@@ -48,12 +48,21 @@ public void epollAvailableOnLinux() {
      * on Linux, Windows and OS X.
      */
     @Test
-    public void openSslAvailable() {
+    void openSslAvailable() {
         assumeTrue(osName.startsWith("linux") || osName.startsWith("windows") ||
                    osName.startsWith("macosx") || osName.startsWith("osx"));
         assumeFalse("false".equals(System.getProperty("com.linecorp.armeria.useOpenSsl")));
 
         assertThat(Flags.useOpenSsl()).isTrue();
         assertThat(OpenSsl.isAvailable()).isTrue();
     }
+
+    @Test
+    void dumpOpenSslInfoDoNotThrowStackOverFlowError() {
+        System.setProperty("com.linecorp.armeria.dumpOpenSslInfo", "true");
+        System.setProperty("com.linecorp.armeria.useOpenSsl", "true");
```

Comment:
You don't need to address this review. Just I guess `@SetSystemProperty` in JUnit Pioneer seems cleaner to set a system property.

```java
@SetSystemProperty(key = "some property", value = "new value")
```

https://junit-pioneer.org/docs/system-properties/

Reply:
That's a nice suggestion. Let me used that. Thanks!